manifests/source.pp

```diff
@@ -23,7 +23,8 @@
 #   Specifies a distribution of the Apt repository.
 #
 # @param repos
-#   Specifies a component of the Apt repository.
+#   Specifies a component of the Apt repository, Valid options: main, contrib, non-free, universe, multiverse and restricted.
```

Comment:
Since the list is not fixed, emphasis these are examples:
```suggestion
#   Specifies a component of the Apt repository, e.g.: main, contrib, non-free, universe, multiverse and restricted.
```

Reply:
have changed the approrach so don't think so its valid anymore. Thanks for your review.